app/facades/food_facade.rb

```diff
@@ -0,0 +1,12 @@
+class FoodFacade
+  def self.search(query)
+    food_data = FdcService.search(query)
+    total_result_count = food_data[:totalHits]
+
+    foods = food_data[:foods].map do |food|
+      Food.new(food)
+    end
+
+    {total_hits: total_result_count, top_ten: foods}
```

Comment:
Excellent job of hitting the external api once and only once, grabbing the totalHits value, creating the food object you need, and then building a hash to return to you controller. This is a great pattern!